src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/help.py

```diff
@@ -63,9 +63,27 @@
             type: command
             short-summary: Creates a new database by restoring from a database backup.
             """
+helps['sql db export'] = """
+            type: command
+            short-summary: Exports a database to a bacpac
+
+            examples:
+              - name: Export bacpac using SAS key
+                text: az sql db import -s myserver -n mydatabase -g mygroup -p password -u login --storage-key "mysaskey" --storage-key-type SharedAccessKey --storage-uri https://mystorageaccount.blob.core.windows.net/bacpacs/mybacpac.bacpac
```

Comment:
can you write the command used to generate the sas key? and make the sas key realistic-looking instead of "mysaskey"